### PR TITLE
Gray loading slots

### PR DESCRIPTION
**Pull request by plumsoftware**

### CODE REVIEW

File: tabletApp/features/roomInfo/src/commonMain/kotlin/band/effective/office/tablet/ui/mainScreen/slotComponent/store/SlotStoreFactory.kt [hunk 1]
```diff
@@ -240,6 +243,32 @@ class SlotStoreFactory(
...
+                    var mainSlot: SlotUi.MultiSlot? = null
+                    val uiSlot = allSlots.firstOrNull { it.slot == intent.slot }
+                        ?: allSlots.mapNotNull { (it as? SlotUi.MultiSlot)?.subSlots }.flatten()
+                            .firstOrNull { it.slot == intent.slot }
+                            ?.apply {
+                                mainSlot = allSlots.mapNotNull { it as? SlotUi.MultiSlot }
+                                    .first { it.subSlots.contains(this) }
+                            }
+
+                    val indexInMultiSlot = mainSlot!!.subSlots.indexOf(uiSlot)
```
Comment (UserNameMax) [inline tabletApp/features/roomInfo/src/commonMain/kotlin/band/effective/office/tablet/ui/mainScreen/slotComponent/store/SlotStoreFactory.kt:258]: What with null safety?

**Review by UserNameMax — CHANGES_REQUESTED** [review]

**Review by UserNameMax — CHANGES_REQUESTED** [review]
Application not build. Build your code before push next time

**Comment by plumsoftware** [comment]
You can see a preview in [card](https://www.notion.so/effectiveband/f3920d143a8d4ac0a8315b6722775f22?v=1b85aa1792db4c53926e27c36d888ea1)

File: tabletApp/features/roomInfo/src/commonMain/kotlin/band/effective/office/tablet/ui/mainScreen/slotComponent/SlotView.kt [hunk 2]
```diff
@@ -208,6 +213,34 @@ private fun CommonSlotView(
...
+) {
+    val slot = slotUi.slot
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Column {
+            Text(
+                text = "Загрузка слота на время: ${slot.start.toFormattedString("HH:mm")} - ${slot.finish.toFormattedString("HH:mm")}", //TODO() replace this hard str
```
Comment (UserNameMax) [inline tabletApp/features/roomInfo/src/commonMain/kotlin/band/effective/office/tablet/ui/mainScreen/slotComponent/SlotView.kt:230]: remove todo

File: tabletApp/features/roomInfo/src/commonMain/kotlin/band/effective/office/tablet/ui/mainScreen/slotComponent/store/SlotStoreFactory.kt [hunk 3]
```diff
@@ -266,6 +279,10 @@ class SlotStoreFactory(
             is SlotUi.MultiSlot -> openMultislot(this, state)
             is SlotUi.SimpleSlot -> slot.execute(state)
             is SlotUi.NestedSlot -> slot.execute(state)
+            is SlotUi.LoadingSlot -> {
+//                Log.i("TAG", state.slots.toString())
```
Comment (UserNameMax) [inline tabletApp/features/roomInfo/src/commonMain/kotlin/band/effective/office/tablet/ui/mainScreen/slotComponent/store/SlotStoreFactory.kt:283]: clean code

**Review by UserNameMax — APPROVED** [review]
remove comment code

**Review by Red1tum — APPROVED** [review]

**Pull request merged**